### DISCUSSION TITLE
Help with ECG Viewer & does it handle fused imaging like PETCT

### DISCUSSION THREAD

**Discussion by Vik1922** [nroduit/weasis-dicom-tools#22] [Q&A]
Using Windows 11 OS
Installed weasis from the Microsoft store.
****
**Q1 -ECG**
How to use ECG viewing tool in Weasis? 
What should be the 'input data set' -JPG or a proprietary data set, please specify what type of files it accepts.
could not find a ECG option in the menu. 
if it is a separate plug in how do i install it 
****
**Q2**
Can weasis fuse & display PET-CT images - i saw in the notes that it can display SUV's 
how do we use the SUV option?

also can it do an MPR on a NM Tomo series? tried but it did not work

thanks in advance.

**Comment by nroduit** [comment]
Please post on the [Weasis](https://github.com/nroduit/Weasis) project and make one post per topic to allow a comprehensive follow-up.

For SUVs, there is already an [answer](https://github.com/nroduit/Weasis/issues/244).

**Comment by Vik1922** [comment]
thanks bro, done!
